pkg/segment/utils/segconsts.go

```diff
@@ -66,7 +66,7 @@ const WIP_SIZE = 2_000_000
 const PQMR_SIZE uint = 4000 // init size of pqs bitset
 const WIP_NUM_RECS = 4000
 const BLOOM_SIZE_HISTORY = 5 // number of entries to analyze to get next block's bloom size
-const BLOCK_BLOOM_SIZE = 200 // the default should be on the smaller side. Let dynamic bloom sizing fix the optimal one
+const BLOCK_BLOOM_SIZE = 10
```

Comment:
it is there just as placeholder, we do bloom creation at the end of wip now where we now the size to allocate for bloom. This var is here just for some value to initialize the bloom filter with, since there is some code that still relies on bi.BF != nil. Eventually we will remove the "double loading" of the bloom variable name